app/main.py

```diff
@@ -1,3 +1,49 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    km: int
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | Distance) -> Distance:
+        if isinstance(other, Distance) is True:
```

Comment:
is True is redundant
```suggestion
        if isinstance(other, Distance):
```

Reply:
Done